### DISCUSSION TITLE
请问openclash默认访问地址的IP在哪里修改呢？

### DISCUSSION THREAD

**Discussion by Live2000** [vernesong/OpenClash#4115] [Q&A]
我的网卡接口绑定了两个不同的IP（其中一个不联网），安装openclash以后，设置界面识别到两个IP，并且把两个IP放在一起了，如下图：
![001](https://github.com/user-attachments/assets/76a86c71-54de-46f8-bcf2-ef8767aed7d7)
![002](https://github.com/user-attachments/assets/79164b71-9422-4a5a-8787-f72f6c7c61d7)

内核日志也会报如下错误：
2024-10-13 18:12:22 level=warning msg="[TCP] dial DIRECT (match Match/) 192.168.1.254:34364(curl) --&gt; 192.168.1.254192.168.100.254:9090 error: dns resolve failed: couldn't find ip"

尝试在外部控制页面，“管理页面公网域名”选项填入了正确的IP，但是不起作用，请问要在什么位置修改呢，谢谢

**Comment by vernesong** [comment]
![image](https://github.com/user-attachments/assets/5992c7a7-a25d-4e81-8a23-764d811bf118)

**Reply by paysenyu** [reply]
我也是同样的问题

这个已经指定了， 10.105.1.1 和 10.105.2.1 都是同一个LAN的物理口 
<img width="414" alt="image" src="https://github.com/user-attachments/assets/1f24d22b-2655-4fc8-a17c-f21b95c587a8">

现在看起来他是把这两个网段做了一个拼接， 合并成了一个IP。      即10.105.1.110.105.2.1
